### ISSUE TITLE
ITMS-90535 ITMS-90166 Error uploading to App Store

### ISSUE THREAD

**Issue by LeeroyDing** [zhangao0086/DKImagePickerController#524]
I encountered these two errors while I submitted an app built with this framework with Carthage.

```
[iTMSTransporter] [$TIMESTAMP] <main> ERROR: ERROR ITMS-90166: "Missing Code Signing Entitlements. No entitlements found in bundle 'com.zhangao0086.DKPhotoGalleryBundle' for executable 'Payload/$PRODUCT_NAME.app/Frameworks/DKPhotoGallery.framework/DKPhotoGallery.bundle/DKPhotoGallery'.""
```

```
[iTMSTransporter] [$TIMESTAMP] <main> ERROR: ERROR ITMS-90535: "Unexpected CFBundleExecutable Key. The bundle at 'Payload/$PRODUCT_NAME.app/Frameworks/DKImagePickerController.framework/DKImagePickerController.bundle' does not contain a bundle executable. If this bundle intentionally does not contain an executable, consider removing the CFBundleExecutable key from its Info.plist and using a CFBundlePackageType of BNDL. If this bundle is part of a third-party framework, consider contacting the developer of the framework for an update to address this issue."
```

**Comment by zhangao0086** [comment]
Fixed in 4.0.4.

**Comment by LeeroyDing** [comment]
It kept not uploading on our production Jenkins and I had to switch away. Closing this issue.
